lib/cc/cli/init.rb

```diff
@@ -8,7 +8,7 @@ class Init < Command
       include CC::Analyzer
 
       def run
-        if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+        if !upgrade? && filesystem.exist?(CODECLIMATE_YAML) && !force
```

Comment:
This is some fairly confusing branching behavior. And I notice that `create_default_configs` is called in two places, here in `run` and lower at the end of `generate_config`. You might be able to make this clearer if:

* `create_default_config` were pulled out of `generate_config` and put somewhere inside of `run`, to more clearly communicate the fact that it's going to get called whether or not `.codeclimate.yml` already exists
* `create_default_config` were renamed to indicate we're talking about engine-specific configs